test/transform/noStrictTest.js

```diff
@@ -16,12 +16,14 @@ describe('Removal of "use strict"', () => {
     );
 
     expectTransform(
-      'foo();\n' +
-      '"use strict";\n' +
-      'bar();'
+      'function foo() {\n' +
```

Comment:
The only test I changed. According to the spec, the original "use strict" is just a string literal. So, I move it into the function body to make it a strict mode for function.